k8s/external-secrets/telegraf.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kubernetes-client.io/v1
```

Comment:
is there a reason to create a separate external-secrets subdir instead of using the more generic workloads subdir?

Reply:
it felt like the secrets were more like namespaces than anything else, so copied that pattern.  Either would work I think.  What makes more sense to you?

Reply:
I like this suggestion, so I'm going to go with it.  thank you @cmharlow

Reply:
Thanks, though I could see either way. Wasn't sure if this indicated some longer term thinking about end to end secrets management or similar. If not, yeah, I prefer (slightly) reusing more-generic folders for now.